xbmc/games/addons/GameClient.h

```diff
@@ -161,6 +162,9 @@ class CGameClient : public ADDON::CAddonDll
   std::unique_ptr<IGameClientPlayback> m_playback; // Interface to control playback
   GAME_REGION           m_region;              // Region of the loaded game
 
+  // In-game saves
+  std::unique_ptr<CGameClientInGameSaves> m_inGameSaves;
```

Comment:
I tried that at first, but CGameClientInGameSaves needs a valid pointer to the dll struct which is initialized only in CGameClient::Initialize so I have to delay the initialization.